msu/utilities/misc.nut

```diff
@@ -0,0 +1,12 @@
+::MSU.isNull <- function( _object )
+{
+	if (_object == null)
```

Comment:
I'd be ok if some of this code is one-lined. For example `if (_object == null) return true;`